src/HotChocolate/Fusion-vnext/src/Fusion.Composition/Logging/LogEntry.cs

```diff
@@ -0,0 +1,68 @@
+using HotChocolate.Skimmed;
+
+namespace HotChocolate.Fusion.Logging;
+
+/// <summary>
+/// Represents an entry in a composition log that describes an issue encountered during the
+/// composition process.
+/// </summary>
+public sealed record LogEntry
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="LogEntry"/> record with the specified values.
+    /// </summary>
+    public LogEntry(
+        string message,
+        string code,
+        LogSeverity severity = LogSeverity.Error,
+        SchemaCoordinate? coordinate = null,
+        ITypeSystemMemberDefinition? member = null,
+        SchemaDefinition? schema = null,
+        object? extension = null)
+    {
+        Message = message;
```

Comment:
Should there be null checks here, or do the static null checks suffice?

Reply:
I think so. I copied this code from v1, and didn't notice this. Thanks.